server/app/views/admin/ti/TrustedIntermediaryGroupListView.java

```diff
@@ -72,10 +82,30 @@ public Content render(List<TrustedIntermediaryGroupModel> tis, Http.Request requ
     return layout.renderCentered(htmlBundle);
   }
 
+  private DivTag renderTiSortSelect(List<QuestionSortOption> sortOptions) {
```

Comment:
I still need to change QuestionSortOption to SortOption to reflect the more generic nature of the enums listed in the file. I am going to do so in my next PR, would prefer to keep this PR focused on adding the new sort selector.